index.html

```diff
@@ -40,18 +40,19 @@ <h1>Debian/Ubuntu Packages</h1>
 
                     <p>To allow the system to check the packages authenticity, you need to provide the release key.</p>
                     <pre># Add the release PGP keys:
-<b>sudo curl -o /usr/share/keyrings/syncthing-archive-keyring.gpg https://syncthing.net/release-key.gpg</b></pre>
+<b>sudo sh -c "test -d /etc/apt/keyrings||mkdir -p /etc/apt/keyrings"
```

Comment:
No need for a test, `mkdir -p` will simply not do anything if the directory already exists.

```suggestion
<b>sudo mkdir -p /etc/apt/keyrings
```